### PR TITLE
add in serializing check in auto annotation model runner

### PR DESCRIPTION
**Pull request by benhoff**

### CODE REVIEW

File: utils/auto_annotation/README.md [hunk 1]
```diff
@@ -50,3 +50,16 @@ $ python cvat/utils/auto_annotation/run_modely.py --py /path/to/python/interp.py
                                                   --show-images
                                                   --show-image-delay 2000
 ```
+
+Visualization isn't always enough. The CVAT has a serialization step that can throw confusing errors on model upload.
+You must install the necessary packages installed, but then you can add the `--serialize` command to ensure that your 
+results will serialize correctly.
+
+```shell
+$ python cvat/utils/auto_annotation/run_modely.py --py /path/to/python/interp.py \
```
Comment (bsekachev) [inline utils/auto_annotation/README.md]: Probably typos is here: run_model``y``.py

File: utils/auto_annotation/README.md [hunk 2]
```diff
@@ -50,3 +50,16 @@ $ python cvat/utils/auto_annotation/run_modely.py --py /path/to/python/interp.py
                                                   --show-images
                                                   --show-image-delay 2000
 ```
+
+Visualization isn't always enough. The CVAT has a serialization step that can throw confusing errors on model upload.
```
Comment (nmanovic) [inline utils/auto_annotation/README.md]: @benhoff , why not change the error message on the server side? Make it more friendly and avoid the extra code in the script?

**Comment by nmanovic** [comment]
@benhoff , my feedback on the change is simple. If it is possible to improve the server and provide more friendly message to users I will follow the way.  The patch tries to workaround a problem instead of fixing it.

**Comment by benhoff** [comment]
@nmanovic I agree the server needs to throw error messages a little bit nicer. I think the `LabeledDataSerializer` is particularly bad at throwing messages that are hard to decipher.

I disagree that this patch works around a problem. It fixes a problem with the script, which is the auto annotation runner does not currently accurately test/run Auto Annotation scripts. As proof, I actually got the Security Segmentation script to visualize completely, only to have the script crash on upload to the server due to a `LabeledDataSerializer` issue. The problem was I was using numpy types. The cv2 module and runner script are happy using numpy types for the visualization. The `LabeledDataSerializer` in the CVAT server, however does not allow numpy types.

Without putting the `LabeledDataSerializer` in the script somewhere, it's impossible to continue to iteratively develop scripts that face these interface issues.

I develop using vim, so right now to spin up an actual CVAT server for debugging requires me to use a tmux session with me manually launching both rqworkers, a redis server, and the CVAT server itself. It's a pain. The web UI is also difficult to use for quick, iterative development (the original reason for the script) required to develop this style of scripts.

I would gladly raise an issue to improve the error handling for the server on `LabeledDataSerializer` errors. That is however a separate level of effort from this pull request.

File: utils/auto_annotation/README.md (hunk 2 above)
Comment (benhoff) [reply utils/auto_annotation/README.md]: See my below response, but I've personally find it difficult to iteratively develop issues with these scripts with the web interface.

File: utils/auto_annotation/README.md (hunk 1 above)
Comment (benhoff) [reply utils/auto_annotation/README.md]: @bsekachev nice catch, fixed!

File: utils/auto_annotation/README.md (hunk 2 above)
Comment (benhoff) [reply utils/auto_annotation/README.md]: @nmanovic I also tweaked the verbiage to more accurately reflect why you would need/want to do this.

**Comment by nmanovic** [comment]
@benhoff , one more time thanks for all contributions. I agree that if `--serialize` option simplifies the debugging process it should be added.

**Pull request merged**